debian/changelog

```diff
@@ -1,3 +1,9 @@
+libfabric (1.11.1rc1-1) unstable; urgency=medium
```

Comment:
```suggestion
libfabric (1.11.1~rc1-1) unstable; urgency=medium
```

The `~` is significant in that it tells `apt` that a version with a `~` is < a version without, so that, in this case, 1.11.1 will be seen as an upgrade to 1.11.1~*.